changelog.rst

```diff
@@ -418,3 +464,17 @@ Improvements:
 .. _`Stuart Quin`: https://github.com/stuartquin
 .. _`Tamas Boros`: https://github.com/TamasNo1
 .. _`Jacek Wielemborek`: https://github.com/d33tah
+.. _`Rodrigo Ramírez Norambuena`: https://github.com/roramirez
+.. _`Anthony Lai`: https://github.com/AnyBucket
```

Comment:
Thanks for doing this @amjith, super excited for the release!
My GH is https://github.com/ajlai though, can we change it in this line? Thanks!